assignment.js

```diff
@@ -66,6 +72,15 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const numbers = [3, 48, 29, 12, 18]
```

Comment:
@annayensietopping nice work! I like the use of the`currentLargestNumber` variable and a conditional to keep track of the current largest number. 

I used an alternative approach that implemented the `sort()` method - see [more details here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L86)